src/platforms/stlink/platform.h

```diff
@@ -61,7 +61,12 @@ extern bool debug_bmp;
 #define NRST_PORT      GPIOB
 #define NRST_PIN_V1    GPIO1
 #define NRST_PIN_V2    GPIO0
+#ifdef SWIM_NRST_AS_UART
+#pragma message "NRST pin is now PB0"
```

Comment:
Think this would be better noted in the documentation - people using a pre-made build of the firmware in this configuration won't see this, and this will hammer the compiler output with this message for every compilation unit that platform.h is included into.

Reply:
where would be the best place in documentation to mention this?

Reply:
We'd think in the platform README.md around where you had to change the wording for what the Makefile flag does - mentioning around there that the nRST signal is remapped to PB0 would be sufficient.